charts/datadog/values.yaml

```diff
@@ -752,6 +752,27 @@ datadog:
         # datadog.securityAgent.runtime.securityProfile.enabled -- Set to true to enable CWS runtime anomaly detection
         enabled: false
 
+        anomalyDetection:
+          # datadog.securityAgent.runtime.securityProfile.anomalyDetection.defaultMinimumStablePeriod -- Set the minimum amount of time to set a security-profile as stable
+          defaultMinimumStablePeriod: 6h
+
+          minimumStablePeriod:
+            # datadog.securityAgent.runtime.securityProfile.anomalyDetection.minimumStablePeriod.exec -- Set the minimum amount of time to set an exec events as stable
+            exec: 6h
+
+            # datadog.securityAgent.runtime.securityProfile.anomalyDetection.minimumStablePeriod.dns -- Set the minimum amount of time to set a dns events as stable
+            dns: 25h
+
+          rateLimiter:
+            # datadog.securityAgent.runtime.securityProfile.anomalyDetection.rateLimiter.period -- Set the number of anomaly detection events allowed per duration
```

Comment:
Changing the comment to something like `Set the rate limiter period of anomaly detection events` would make more sense here.

Reply:
@YoannGh thanks, I'll updated